MiniSim/Service/DeviceService.swift

```diff
@@ -389,7 +472,40 @@ extension DeviceService {
 
   static func getAndroidDevices() throws -> [Device] {
     Thread.assertBackgroundThread()
+    let emulators = try getAndroidEmulators()
+    let devices = try getAndroidPhysicalDevices()
+    return emulators + devices
+  }
+
+  static func getAndroidPhysicalDevices() throws -> [Device] {
+      let adbPath = try ADB.getAdbPath()
+      let output = try shellOut(to: adbPath, arguments: ["devices", "-l"])
+      var splitted = output.components(separatedBy: "\n")
+      splitted.removeFirst() // removes 'List of devices attached'
+      let filtered = splitted.filter { !$0.hasPrefix("emulator-") }
```

Comment:
Maybe we can filter this out instead of always assuming this is at the first place?

Reply:
```suggestion
      splitted.removeFirst() // removes 'List of devices attached'
      let filtered = splitted.filter { !$0.contains("emulator") }
```
Like this? Yeah thats fine for me too

Reply:
```suggestion
      splitted.removeFirst() // removes 'List of devices attached'
      let filtered = splitted.filter { !$0.contains("emulator") }
```

Reply:
done